### PR TITLE
Artp 318/launch limit checker

### PR DESCRIPTION
**Pull request by snony**
**Feature**
- User should be able to download and launch Limit Checker.

**Solution**
- Added a config file with the download link of Limit Checker.
- Downloading and Launching Limit Checker works on RT OpenFin. However, you need to download it first, and then click the download button again and it will launch it.
- While in RT browser, it only downloads the Limit Checker. There is no way to launch it due to Browser security issue.

### CODE REVIEW

File: src/client/src/routes/SimpleLauncher/tools.ts [hunk 1]
```diff
@@ -13,10 +13,13 @@ export async function open(config: ConfigType): Promise<Window | fin.OpenFinWind
       switch (provider.as) {
         case 'window':
           return createOpenFinWindow(config)
+        case 'download':
+          //either download LimitChecker or launches it if it exists.
```
Comment (danmcdaid) [inline src/client/src/routes/SimpleLauncher/tools.ts]: is this comment neccesary?

File: src/client/src/routes/SimpleLauncher/tools.ts [hunk 2]
```diff
@@ -85,3 +86,31 @@ export function createOpenFinWindow({ name, url, provider: { options } }: Config
     )
   })
 }
+
```
Comment (danmcdaid) [inline src/client/src/routes/SimpleLauncher/tools.ts]: feels to me like single responsibility violation and `tools.ts` is starting to do too much (arguably was before this addition).  I'd consider at this point splitting out these handlers?

Comment (snony) [reply src/client/src/routes/SimpleLauncher/tools.ts]: I am going to leave it as is and will address this comment in the subsequent PR.

File: src/client/src/routes/SimpleLauncher/tools.ts (hunk 1 above)
Comment (snony) [reply src/client/src/routes/SimpleLauncher/tools.ts]: Yes, it is not obvious to a user.

**Pull request merged**